src/morse/index.js

```diff
@@ -58,6 +58,53 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) 
+{
+  //check to ensure the input text is a valid else throw an error
+  try
+  {
+    if(typeof text !== "string") 
+      throw `Please provide a morse string`;
 
-module.exports = morse;
+    if(text === undefined) 
+      throw `Please provide a morse string`;      
+  }
+  catch(err)
+  {
+    return err;
+  }
```

Comment:
Wrapping it `try..catch` means the error is thrown, catch and the function returns an error instead of throwing one.

Reply:
okay sir